src/components/MenuTransition.svelte

```diff
@@ -0,0 +1,21 @@
+<!--
+  (c) 2023, Center for Computational Thinking and Design at Aarhus University and contributors
+
+  SPDX-License-Identifier: MIT
+ -->
+
+<script lang="ts">
+  import Transition from 'svelte-transition';
+  export let show: boolean;
+</script>
+
+<Transition
```

Comment:
This is exactly as per svelte-headless's docs for the menu. I tried and failed to customise it to look a bit more like the other Foundation apps. Seems good enough to me!